### PR TITLE
[DOC-7773] Trimming CodeStream UI docs 

### PR DESCRIPTION
**Pull request by urbiz-nr**
See DOC-7773 for more details.

On this commit:
* Setting new structure 
* Moving sidebar and shortcuts docs

### CODE REVIEW

**Comment by planteater** [comment]
@urbiz-nr - One thing I noticed throughout the docs is that images with captions are left-aligned, while all other images are centered. See the the "Discuss code" and "Request feedback" articles for examples.

**Comment by planteater** [comment]
I think including Permalinks in the CodeStream UI Overview article is probably too much of a force fit. In reality, a permalink is another flavor of codemark and we should probably include them in the Discuss Code article. My suggestion is to move it there, but in doing so kill of the "Permalinks for sharing anywhere" section. That's effectively covered by the bullet in the Discuss Code > Manage Codemarks section. 

Permalinks are a little used feature, so maybe including them under Discuss Code > Advanced Features is a good way to go.

File: src/content/docs/codestream/codestream-settings/team-administration.mdx [hunk 1]
```diff
@@ -1,64 +1,67 @@
...
+Click **Add mapping**, enter your Git email address, and then select your entry from the list of organization members. That way, when someone comments on code committed by dave@webmail.com, CodeStream will know to at-mention you (such as, `dave@acme.com`).
+
+![A screenshot showing a blame map.](./images/MyOrgBlameMap.png "blame-map")
+
+While non-admins can only create blame maps for themselves, admins can create blame maps for any member of the organization. This is useful for reassigning code ownership when people leave the organization.
 
 ## Organization settings [#organization-settings]
 
-If you're an organization admin, look for the **Organization Admin** menu under the headshot menu at the top of the CodeStream pane.
+If you're an admin, use the **Organization Admin** menu under the headshot menu at the top of the CodeStream pane to change your organization's name, manage [onboarding](#onboarding) and [feedback request](#feedback-request) settings, [export data](#export), or delete your org.
```
Comment (planteater) [inline src/content/docs/codestream/codestream-settings/team-administration.mdx:42]: The "feedback request" link doesn't appear to be working. I think the anchor link is missing an "s" at the end.

File: src/content/docs/codestream/how-use-codestream/discuss-code.mdx [hunk 2]
```diff
@@ -3,11 +3,12 @@ title: Discuss code on CodeStream
...
 ---
 
 ## What is a codemark? [#codemark]
 
-Quite simply, a codemark is a discussion connected to the code. It could be a question, a suggestion, a bug report, or documentation. All of these discussions are saved, anchored to the blocks of code they refer to, so that they can be leveraged in the future. It could be a new developer joining the team, a developer trying to fix a bug in someone else’s code, or even just you trying to remember why you made that change six months ago. Whatever the case, CodeStream helps you understand the code by surfacing the discussions in a contextual way.
+Quite simply, a codemark is a [discussion](/docs/codestream/how-use-codestream/discuss-code) connected to the code. It could be a question, a suggestion, a bug report, or documentation. All of these discussions are saved, anchored to the blocks of code they refer to, so that they can be leveraged in the future. It could be a new developer joining the team, a developer trying to fix a bug in someone else’s code, or even just you trying to remember why you made that change six months ago. Whatever the case, CodeStream helps you understand the code by surfacing the discussions in a contextual way.
```
Comment (planteater) [inline src/content/docs/codestream/how-use-codestream/discuss-code.mdx:11]: Need to remove the "discussion" link, as it's self referential.

File: src/content/docs/codestream/codestream-settings/team-administration.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/codestream/codestream-settings/team-administration.mdx:42]: Good catch, thanks!

File: src/content/docs/codestream/how-use-codestream/discuss-code.mdx (hunk 2 above)
Comment (urbiz-nr) [reply src/content/docs/codestream/how-use-codestream/discuss-code.mdx:11]: Right, removed!

File: src/content/docs/codestream/how-use-codestream/start-work.mdx [hunk 3]
```diff
@@ -1,34 +1,74 @@
...
-Click an issue to get started working on it.
+Click an issue to get started working on it. 
 
 ![A screenshot showing how to begin work on an issue](/images/StartWork2.png "start work")
 
+You can create a feature branch, update the ticket status, and even update your status on Slack.
+
 If your team uses a feature branch Git workflow, you can create a branch to do your work on. Use the dropdowns to either change the base branch or edit the name of the feature branch.
 
 ![A screenshot showing how to edit a branch name](/images/StartWork-BranchOptions.png "branch options"))
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/codestream/how-use-codestream/start-work.mdx:46]: Thanks!

**Comment by urbiz-nr** [comment]
> @urbiz-nr - One thing I noticed throughout the docs is that images with captions are left-aligned, while all other images are centered. See the the "Discuss code" and "Request feedback" articles for examples.

I know, current docs are like this. Was planning to look into it with your feedback.

File: src/content/docs/codestream/how-use-codestream/discuss-code.mdx [hunk 4]
```diff
@@ -177,4 +210,26 @@ Click the ellipses menu for any codemark and you'll see options to manage the co
...
\ No newline at end of file
+* **Reposition codemark**: In most cases, a codemark will automatically remain linked to the block of code it refers to as the file changes over time. For example, if you cut the block of code and paste it at a different location in the file, the codemark will move right along with it. There are some scenarios, however, that CodeStream isn't able to handle automatically. For example, if you pasted the block of code into a different file. In these cases, the **Reposition** codemark allows you to select the new location of the block of code so that the codemark is displayed properly.
+
+## Permalinks for sharing anywhere [#permalinks]
```
Comment (planteater) [inline src/content/docs/codestream/how-use-codestream/discuss-code.mdx:215]: The "Copy link" bullet in the list above now needs its anchor link updated to point to https://docswebsitedevelop-doc7773consolidatecodestreamui.gtsb.io/docs/codestream/how-use-codestream/discuss-code/#permalinks

**Comment by planteater** [comment]
In the account-settings article, I just noticed the following line:

"If you'd like to cancel your CodeStream account, find that option under Other Actions."

That line has probably been out of date for some time now. To delete your account, there's a red button at the bottom of your profile.

Screen Shot 2022-02-21 at 9 36 48 AM

File: src/content/docs/codestream/how-use-codestream/discuss-code.mdx (hunk 4 above)
Comment (urbiz-nr) [reply src/content/docs/codestream/how-use-codestream/discuss-code.mdx:215]: Done!

**Review by paperclypse — APPROVED** [review]
I like these changes a lot! Really excellent work, @urbiz-nr!

**Pull request merged**